bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubLiveDeviceData.java

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.neohub.internal;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * A wrapper around the JSON response to the JSON GET_LIVE_DATA request
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class NeoHubLiveDeviceData extends NeoHubAbstractDeviceData {
+
+    private static final Gson GSON = new Gson();
+
+    @Nullable
+    @SerializedName("TIMESTAMP_ENGINEERS")
+    private BigDecimal timestampEngineers;
+    @Nullable
+    @SerializedName("TIMESTAMP_SYSTEM")
+    private BigDecimal timestampSystem;
```

Comment:
```suggestion
    @SerializedName("TIMESTAMP_ENGINEERS")
    private @Nullable BigDecimal timestampEngineers;
    
    @SerializedName("TIMESTAMP_SYSTEM")
    private @Nullable BigDecimal timestampSystem;
```

You get the idea, please update elsewhere as applicable.